<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClutterUI</class>
 <widget class="QWidget" name="ClutterUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clutter Base</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
   <item>
    <widget class="QTabWidget" name="db_view">
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="table_tab">
      <attribute name="title">
       <string>Table View</string>
      </attribute>
      <layout class="QVBoxLayout" name="db_layout"/>
     </widget>
     <widget class="QWidget" name="view_tab">
      <attribute name="title">
       <string>Grid View</string>
      </attribute>
      <layout class="QVBoxLayout" name="view_tab_layout"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controls_gb">
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="run_query_button">
        <property name="text">
         <string>Run Query</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="query_text"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Display Images</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QCheckBox" name="display_persp">
           <property name="text">
            <string>Persp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="display_front">
           <property name="text">
            <string>front</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="display_side">
           <property name="text">
            <string>Side</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="display_top">
           <property name="text">
            <string>top</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>DB Controls</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QPushButton" name="delete_from_db">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="select_db">
           <property name="text">
            <string>Select DB</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="insert_to_db">
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="new_db">
           <property name="text">
            <string>New DB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
